group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinator.java

```diff
@@ -164,5 +166,20 @@ CompletableFuture<List<OffsetFetchResponseData.OffsetFetchResponseTopics>> fetch
         String groupId,
         boolean requireStable
     );
+
+    /**
+     * Commit offsets for a given Group.
```

Comment:
are the descriptions for `fetchOffsets(), fetchAllOffsets(), commitOffsets()` "Group" instead of "Generic Group" since they can apply to both Generic and Consumer groups? just noticed the difference in join/sync/leave group.

Reply:
Yes, that's right.